Doppler.HtmlEditorApi/Domain/CampaignState.cs

```diff
@@ -12,4 +27,18 @@ public record CampaignState(bool OwnCampaignExists, bool ContentExists, int? Edi
         });
     public bool IsWritable => CampaignStatus.HasValue && WritableStatus.Contains(CampaignStatus.Value);
 }
-public record NoExistCampaignState() : CampaignState(false, false, null, null);
+public record NoExistCampaignState() : CampaignState(false, false, null, null, TestABCondition.TypeClassic, null, null, null);
+public record ClassicCampaignState(
+    bool ContentExists,
+    int? EditorType,
+    CampaignStatus?
+    CampaignStatus) : CampaignState(true, ContentExists, EditorType, CampaignStatus, TestABCondition.TypeClassic, null, null, null);
```

Comment:
Tendría sentido? para simplificar el código? creo que no tiene sentido hacerlo en este PR, pero luego se podría considerar esta opción.

```suggestion
public record ClassicCampaignState(
    int idCampaign,
    bool ContentExists,
    int? EditorType,
    CampaignStatus?
    CampaignStatus) : CampaignState(true, ContentExists, EditorType, CampaignStatus, TestABCondition.TypeClassic, idCampaign, null, null);
```

Reply:
Ok, lo puedo dejar como un TODO

Reply:
No tengo una opinión marcada acerca de esto 😞